neo/app/controllers/Application.scala

```diff
@@ -59,7 +60,8 @@ class Application @Inject()(val wsClient: WSClient,
   }
 
   def ws = WebSocket.accept[JsValue, JsValue] { request =>
-    ActorFlow.actorRef(out => NeoActor.props(out, wsClient, request.host, config))
+    val prop = BerryClient.props(new JSONParser(), manager, new QueryPlanner(), config)
+    ActorFlow.actorRef(out => RequestRouter.props(out, prop, config))
```

Comment:
So we pass `out` into `Berry` and in `RequestRouter`, we only send the request to `berry` and let `berry` itself deal with sending back the response?